scripts/photon-settings.sh

```diff
@@ -36,16 +36,15 @@ tdnf install -y \
   httpd
 
 echo '> Adding K8s Repo'
-curl -L https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg -o /etc/pki/rpm-gpg/GOOGLE-RPM-GPG-KEY
-rpm --import /etc/pki/rpm-gpg/GOOGLE-RPM-GPG-KEY
+K8S_PACKAGE_REPO_VERSION=$(jq -r < ${VEBA_BOM_FILE} '.["kubernetes"].packageRepoTag')
```

Comment:
Replace this with following (which uses string expansion to strip away anything from the last period)

```
K8S_PACKAGE_REPO_VERSION=${$(jq -r < ~/git/vcenter-event-broker-appliance/veba-bom.json '.["kubernetes"].gitRepoTag')%.*}
```

Reply:
It's now:
```shell
K8S_PACKAGE_REPO_VERSION_FULL=$(jq -r < "${VEBA_BOM_FILE}" '.["kubernetes"].gitRepoTag')
K8S_PACKAGE_REPO_VERSION=${K8S_PACKAGE_REPO_VERSION_FULL%.*}
```